modules/fedimint-ln-client/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-mod db;
+pub mod db;
```

Comment:
I'm ok with making this pub, but is there a reason for migration tests not to live in their respective crate (client or server)? We mainly have the tests crates for integration-testing client and server together.

Reply:
I originally did it just so all of the migration tests were in the same spot, but now I'm somewhat regretting it as it made me change all of these things to pub. Would you prefer if I moved them back? Probably best done in a separate PR

Reply:
I agree it's not worth uprooting this PR. Let's try to tidy up public interfaces in a separate PR. Not super high priority.

Reply:
Will leave this for now and handle in separate PR